Snowplow iOSTests/TestGeneratedJsons.m

```diff
@@ -61,11 +61,13 @@ - (void)tearDown {
     [super tearDown];
 }
 
-- (void) testScreenContextJson {
+/* //$ TODO: To fix as getScreenContextWithScreenState has been removed.
```

Comment:
TODO left in. Maybe need to change this into more "end to end" test to ensure the context is on the event as expected?

Reply:
:( I missed all of these `//$` comments this time!
I agree that this sounds more like an end-to-end test.
However, I've fixed it here using ScreenStateMachine.
And it seems also an elegant solution!